editor/src/components/canvas/canvas-strategies/strategies/keyboard-set-opacity-strategy.tsx

```diff
@@ -57,7 +57,7 @@ export function keyboardSetOpacityStrategy(
         setProperty('always', path, PP.create('style', 'opacity'), inputValue),
       )
 
-      return strategyApplicationResult(commands)
+      return strategyApplicationResult(commands, 'rerender-all-elements')
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.